content/docs/setup/kubernetes/additional-setup/cni/index.md

```diff
@@ -198,27 +151,51 @@ accompanying Istio proxy sidecar by checking that the pod meets all of the follo
-| `traffic.sidecar.istio.io/includeInboundPorts` | `<port1>,<port2>,...` | Pod's list of `containerPorts` | Comma separated list of inbound ports for which traffic is to be redirected to the Istio proxy sidecar.  The value of "*" redirects all ports. |
-| `traffic.sidecar.istio.io/excludeInboundPorts` | `<port1>,<port2>,...` | | Comma separated list of inbound ports to be excluded from redirection to the Istio sidecar proxy.  Only valid when `includeInboundPorts` is "*". |
+| `traffic.sidecar.istio.io/includeOutboundIPRanges` | `<IPCidr1>,<IPCidr2>,...` | `"*"` | Optional comma separated list of IP ranges in CIDR form to redirect to the sidecar proxy.  The default value of `"*"` redirects all traffic. |
+| `traffic.sidecar.istio.io/excludeOutboundIPRanges` | `<IPCidr1>,<IPCidr2>,...` | | Optional comma separated list of IP ranges in CIDR form to be excluded from redirection.  Only applies when `includeOutboundIPRanges` is `"*"`. |
+| `traffic.sidecar.istio.io/includeInboundPorts` | `<port1>,<port2>,...` | Pod's list of `containerPorts` | Comma separated list of inbound ports for which traffic is to be redirected to the Istio proxy sidecar.  The value of `"*"` redirects all ports. |
+| `traffic.sidecar.istio.io/excludeInboundPorts` | `<port1>,<port2>,...` | | Comma separated list of inbound ports to be excluded from redirection to the Istio sidecar proxy.  Only valid when `includeInboundPorts` is `"*"`. |
+| `traffic.sidecar.istio.io/excludeOutboundPorts` | `<port1>,<port2>,...` | | Comma separated list of outbound ports to be excluded from redirection to Envoy. | 
+| `traffic.sidecar.istio.io/kubevirtInterfaces` | `<ethX>,<ethY>,...` | | Comma separated list of virtual interfaces whose inbound traffic (from VM) will be treated as outbound. |
+
+Cluster-wide defaults for these parameters can be set using the corresponding
+[`global.proxy.*` parameters](/docs/reference/config/installation-options/#global-options) when installing the `istio`
+Helm chart.  For instance, to set a default `traffic.sidecar.istio.io/excludeOutboundIPRanges` option for all pods in
+the cluster when installing Istio:
+
+{{< text bash >}}
+$ helm template install/kubernetes/helm/istio --name istio --namespace istio-system \
+    --set global.proxy.excludeOutboundPorts=53 | kubectl apply -f -
+{{< /text >}}
 
 ### Logging
 
-The Istio CNI plugin is run by the container runtime process space, and, therefore, the log entries are added within
+The Istio CNI plugin is run by the container runtime process space. Therefore, the log entries are logged by the
 the `kubelet` process.
 
-## Compatibility with other CNI plugins
+### Compatibility with application init containers
+
+The Istio CNI plugin may cause networking problems for any application `initContainers`, since the plugin
```

Comment:
This entire paragraph is very difficult to follow. What can cause problems? Are the problems caused by my container, by Istio, by CNI? Where do I need to go to fix it? Should I change the sidecar? Those are just some of the questions a reader might have by the end of the paragraph. 
Please rewrite. IMO explaining the process as it works normally first and then pointing out which step can be problematic because the redirect gets put in place before the Istio initialization container runs, would go a long way. Ideally, explain it in a numbered list.

Reply:
> What can cause problems?

I think this is pretty clear: "The Istio CNI plugin may cause networking problems"

Reply:
Split into a numbered list.